recipes/pdfcombine/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "pdfcombine" %}
+{% set version = "0.3.1" %}
+{% set sha256 = "23e3af619a4859008745ac61a6040a230023cd4e5c4c667d9b8ea248726f47ed" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/tdegeus/pdfcombine/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [win and vc<14]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
```

Comment:
Why do you need a compiler?

Reply:
Otherwise CMake complains (on Linux)

Reply:
Why do you need cmake in the first place to copy a file?

Reply:
@isuruf Maybe it is indeed a bit convoluted ☺️. I provide CMake for user to install on all platforms. But since conda already deals with it, I could simply do the copying myself. I will update the PR! Is it possible to do this platform independent directly from `meta.yaml`?